README.md

```diff
@@ -1 +1,3 @@
+Hello Microverse Project
+
 This is simple first project in microverse
```

Comment:
- [ ] Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.